decisionserver/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicTest.java

```diff
@@ -0,0 +1,152 @@
+ */
+
+package org.jboss.test.arquillian.ce.decisionserver;
+
+import static org.junit.Assert.assertEquals;
+
+import org.jboss.arquillian.ce.api.ConfigurationHandle;
+import org.jboss.arquillian.ce.api.ExternalDeployment;
+import org.jboss.arquillian.ce.api.OpenShiftResource;
+import org.jboss.arquillian.ce.api.OpenShiftResources;
+import org.jboss.arquillian.ce.api.RunInPod;
+import org.jboss.arquillian.ce.api.RunInPodDeployment;
+import org.jboss.arquillian.ce.api.Template;
+import org.jboss.arquillian.ce.api.TemplateParameter;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kie.server.api.marshalling.MarshallingFormat;
+import org.kie.server.api.model.KieServerInfo;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.client.KieServicesConfiguration;
+import org.kie.server.client.KieServicesFactory;
+
+/**
+ * @author Filippe Spolti
+ */
+
+@RunWith(Arquillian.class)
+@RunInPod
+@ExternalDeployment
+@Template(url = "https://raw.githubusercontent.com/jboss-openshift/application-templates/master/decisionserver/decisionserver62-basic-s2i.json",
+        labels = "application=kie-app",
+		parameters = {
+        @TemplateParameter(name = "KIE_SERVER_USER", value="kieserver"),
+        @TemplateParameter(name = "KIE_SERVER_PASSWORD", value="Redhat@123")})
+@OpenShiftResources({
+        @OpenShiftResource("classpath:decisionserver-internal-imagestream.json"),
+        @OpenShiftResource("classpath:decisionserver-service-account.json"),
+        @OpenShiftResource("classpath:decisionserver-app-secret.json")
+})
+public class DecisionServerBasicTest {
+  
+    public static final String DECISIONSERVER_ROUTE_HOST = "http://kie-app-%s.router.default.svc.cluster.local/kie-server/services/rest/server/";
+    private static final MarshallingFormat FORMAT = MarshallingFormat.JSON;  
+    private static KieServicesConfiguration kieServicesConfiguration;  
+    private static KieServicesClient kieServicesClient; 
+    
+    //kie-server credentials
+    private static final String USERNAME = "kieserver";
+    private static final String PASSWORD = "Redhat@123";
+
+    @ArquillianResource
+    ConfigurationHandle configuration;
+    
+    @Deployment
+    @RunInPodDeployment
+    public static WebArchive getDeployment() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, "run-in-pod.war");
+        war.setWebXML(new StringAsset("<web-app/>"));
+
+        war.addAsLibraries(
+                Maven.resolver()
+                        .resolve("org.kie:kie-server-client:6.2.0.Final")
```

Comment:
Do we really need to specify the version here?
Afaik, it should grab the version from the pom.